namnder/studienamnden/body.md

```diff
@@ -0,0 +1,89 @@
+
+### Årskursrepresentanter
+Årskursrepresentanter är studenter som representerar sin årskurs gentemot både studenterna och lärarna. De fungerar som en röst för de som inte vill/vågar ta kontakt med lärarna själva. Har du synpunkter gällande kurser, lärare eller liknande så finns din årskursrepresentant där för att hjälpa dig.
+
+
+### Övriga intresserade studenter
+SN är en öppen nämnd, vilket innebär att alla sektionsmedlemmar som vill får engagera sig i nämnden. För mer information, kontakta Studienämndens ordförande (SNO)
+
+### Studienämndens ordförande
+Studienämndens ordförande (SNO) leder arbetet i Studienämnden samt inom studiebevakning i allmänhet.
+Vad ska en student veta om din post?
+Studienämndsordförande är en förtroendevald funktionärspost i Datasektionen och reglementet säger att SNO ska
+verka för att alla sektionsmedlemmar är representerade i Studienämnden
+att synpunkter och förslag på förbättringar från sektionsmedlemmarna samlas in och föras vidare till ansvariga parter
+verka för att nå ut med varför Studienämnden finns och hur man som student kan påverka sin utbildning via oss.  
+ansvara för att arbetet bevaras och dokumenteras så att framtida medlemmar kan följa upp det
+ 
+Vad vill du att en student ska säga till dig?
+Jag vill gärna veta om en student har åsikter om antingen våran utbildning eller Studienämnden, detta för att vi i studienämnden jobbar för att ta vara på den. Dessutom kanske just du kan hjälpa oss att vet om vi arbetar med rätt saker i studierelaterade frågor.
+
+
+### Programansvarig student
+Programansvarig student (PAS) arbetar tillsammans med Studienämndens ordförande för att förbättra utbildningens kvalitet. PAS ansvarar huvudsakligen för kontakten till lärare samt utbildningsansvariga på CSC-skolan.
+
+### Styrelseledamot för utbildningsfrågor
+Styrelseledamot för utbildningsfrågor (D-UF) verkar för en bättre utbildningskvalitet på sektionen. Tanken är att D-UF ska ha en total bild över utbildningen. Vidare ansvarar ledamoten, D-UF, för att kontakten mellan styrelsen och Studienämndens ordförande/ Programansvarig student.
+
+### Verksamhet
+Enligt sektionens reglemente är den huvudsakliga verksamheten att:
+Inhämta teknologernas åsikter om studiesituationen och med dessa som utgångspunkt arbeta för att förbättra kvaliteten på utbildningen.
+Utvärdera information och beslut från KTH:s organ.
+Bevaka och söka förbättra den fysiska och psykosociala studiemiljön.
+Studienämndsmöten
+Studienämndsmöten (SN-möten) hålls minst en gång per period. Mötena är öppna för samtliga intresserade sektionsmedlemmar. Man kan alltid ta upp studierelaterade frågor eller studienämndens verksamhet. Annars diskuterar vi aktuella problem i kurser och funktionärer som SSO (Studerandeskyddsombud), berättar vad de gjort från föregående möte.
+
+[Tidigare mötesprotokoll](hittas https://drive.google.com/drive/folders/0B5oC_KdJBCU7amRQb3RFb3VQbzg)
```

Comment:
Im guessing you mean `Tidigare mötesprotokoll hittas [här](https://drive.google.com/drive/folders/0B5oC_KdJBCU7amRQb3RFb3VQbzg)`